install-guard.sh

```diff
@@ -17,24 +17,24 @@ main() {
 
     get_os_type
     get_latest_release |
-        while read MAJOR_VER; read VERSION; do
-            mkdir -p ~/.guard/$MAJOR_VER ~/.guard/bin ||
+        while read -r MAJOR_VER; read -r VERSION; do
+            mkdir -p ~/".guard/$MAJOR_VER" ~/.guard/bin ||
```

Comment:
Can you tell me why you changed it to `~/".guard/$MAJOR_VER"` instead of `~/.guard/"$MAJOR_VER"` recommended by shellcheck?

Reply:
This is my mistake. I'll fix them.